### ISSUE TITLE
Vgate iCar Pro BT 4.0

### ISSUE THREAD

**Issue by SappiBT** [EVNotify/EVNotify#187]
Bought the Vgate OBD2 dongle.

Works OK, connects to my iPhone (iOS 13) powers off (-30 min) after stopping te car (Hyundai Kona Electric), but never powers up when I start the car again. Specs say it does, but not here. Have to take it out and in again before it works again.

Anybode?

**Comment by SliVee** [comment]
I have same dongle and it powers up just fine when I start he car and I have also Kona EV. I just need to figure out how EVNotify can connect automatically to dongle. It takes usually time and multiple tries to connect.

**Comment by lepinkainen** [comment]
Hyundais (at least my Ioniq) power down the OBD bus after 30 minutes. It does stay on if you are charging though.

**Comment by mhantsch** [comment]
@SliVee EVNotify will try to reconnect to the Dongle if you tap on the Settings button, then on Dashboard button (the buttons on the bottom of the screen). I made an 'Automate' flow for my phone to do that automatically whenever the phone in the car gets power.

**Comment by philos31** [comment]
@mhantsch interesting, could you share that automate flow? I seem to have the same problem that evnotify does not autoconnect to the dongle. So I would be interested in learning how to automate that. Thanks (Danke)

**Comment by mhantsch** [comment]
@philos31 check this out: https://www.goingelectric.de/wiki/EVNotify-Alternative-Loesung-mit-Automate/
(In German, but hopefully Google Translate can help you.)

**Comment by Tchakpoum** [comment]
> 
> 
> Bought the Vgate OBD2 dongle.
> 
> Works OK, connects to my iPhone (iOS 13) powers off (-30 min) after stopping te car (Hyundai Kona Electric), but never powers up when I start the car again. Specs say it does, but not here. Have to take it out and in again before it works again.
> 
> Anybode?

Same for me !

i can configure the dongle whith "Autodoctor" but cant link him on EVnotify and after the dongle shut down, he didnt start again...

I am using IOS and i think its a part of the problem :-/

**Comment by GPlay97** [comment]
Yes. Known issue on iOS. Will get better hopefully with v3. Currently the main reason is also a restriction from Apple since it's unfortunately forbidden for me to run as background task

**Comment by Tchakpoum** [comment]
> 
> 
> Yes. Known issue on iOS. Will get better hopefully with v3. Currently the main reason is also a restriction from Apple since it's unfortunately forbidden for me to run as background task

Apple or the "Downgrade realm" !

when did the V3 is planned ?

Thanks for all btw !
